src/scripting.c

```diff
@@ -1858,7 +1858,7 @@ void ldbSendLogs(void) {
 int ldbStartSession(client *c) {
     ldb.forked = (c->flags & CLIENT_LUA_DEBUG_SYNC) == 0;
     if (ldb.forked) {
-        pid_t cp = redisFork();
+        pid_t cp = redisFork(CHILD_TYPE_LDB);
```

Comment:
@oranagra typo? (LDB -> RDB)

Reply:
no.. if it was a typo it would have failed to build.. 8-)
LDB = Lua Debugger (look at the function we're inside)